src/tree/aksClusterTreeItem.ts

```diff
@@ -18,7 +18,7 @@ export interface AksClusterTreeNode {
 export default class AksClusterTreeItem extends AzureTreeItem implements AksClusterTreeNode {
     constructor(
         parent: AzExtParentTreeItem,
```

Comment:
☕️ let’s discuss about this change so I think by contract the “cloud explorer” guarantee target will always have name and type, hence let’s relive this small change from this and and the if conditional check. Unless you were able to replicate something. 

We could make this property not private in separate Pr and involve Ivan as well. What do you think? Just throwing some ideas.

Reply:
❤️❤️❤️ cool, let’s drop this minor change and I shall merge this good one into the feature branch.

thanks heaps , this is soo nice ❤️😊🙏